distributed/worker.py

```diff
@@ -268,7 +273,9 @@ def heartbeat(self):
                     in_memory=len(self.data),
                     ready=len(self.ready),
                     in_flight=len(self.in_flight_tasks),
-                    **self.monitor.recent())
+                    custom_metrics_names=list(custom_metrics.keys()),
+                    **self.monitor.recent(),
+                    **custom_metrics)
```

Comment:
Should we add custom_metrics to `_register_with_scheduler` too? The reason I am asking is because self.monitor.recent is passed in [there](https://github.com/dask/distributed/pull/2169/files#diff-3f78e3c9a9a81b6355c8e594dce0b8f3R323).

Reply:
Maybe.  It wouldn't hurt.  I don't think it's critical though.